g3/git/client.py

```diff
@@ -0,0 +1,11 @@
+from g3.git.shell import Shell
+
+
+def commit(message: str) -> None:
+    """
+    Commit the current state of the index to the repository.
+
+    :param message: The commit message.
+    """
+    sh = Shell()
+    sh.git("commit", "-m", message)
```

Comment:
We need to support `--edit` option too (edit option after preview). Can you support it exposing an edit boolean in the args?